src/interfaces/widget.ts

```diff
@@ -166,6 +166,7 @@ export interface WidgetOptions {
 }
 
 export interface WidgetMeta {
+    id?: string,
```

Comment:
There is no `id` property in widget meta description.  
There was once, but we deprecated it in favor of `name` property due to `id` being autogenerated on executing sql data sets of widgets, and so was lacking consistency.